src/Couchbase.Lite.Tests.UWP/Couchbase.Lite.Tests.UWP.csproj

```diff
@@ -133,22 +133,22 @@
     <Content Include="Assets\Wide310x150Logo.scale-200.png" />
   </ItemGroup>
   <Choose>
-  <When Condition="'$(Configuration)'=='Debug'">
-    <ItemGroup>
-      <PackageReference Include="Couchbase.Lite.Enterprise.Debug" Version="2.5.0-b0027" />
-    </ItemGroup>
-  </When>
+    <When Condition="'$(Configuration)'=='Debug'">
+      <ItemGroup>
+        <PackageReference Include="Couchbase.Lite.Enterprise.Debug" Version="2.5.0-b0027" />

```

Comment:
This one was not updated.  If you are going to update one it makes sense to update the other (edit by hand)